crescent/client.py

```diff
@@ -82,17 +69,19 @@ class Bot(crescent.Mixin, hikari.GatewayBot):
             command_after_hooks:
                 List of hooks to run after all commands.
         """
-        kwargs["token"] = token
-        super().__init__(**kwargs)
+        active_clients[app] = self
+
+        self._app = app
```

Comment:
Is there a reason this has to be hidden? Seems like it might be useful for `plugin.client.app`

Reply:
I want people to use `plugin.app` over `plugin.client.app`.

Reply:
Alright fair enough

Reply:
I think im going to make it public actually so you can dependency inject client and still have access to app.